stix2validator/validator.py

```diff
@@ -274,9 +302,39 @@ def run_validation(options):
     except NoJSONFileFoundError as e:
         output.error(e.message)
 
-    results = {}
-    for fn in files:
-        results[fn] = validate_file(fn, options)
+    results = [validate_file(fn, options) for fn in files]
```

Comment:
The old `ValidationResults` class stored a filename inside it.  So why store filenames a second time, in dict keys?  That didn't make sense (unless you needed fast lookup by filename... which didn't seem to be the case?), so I turned this into a list.  Same thing applies to the new `FileValidationResults` class.